react/src/services/UserService.js

```diff
@@ -1,7 +1,8 @@
 import ApiService from './ApiService';
 
 const ENDPOINTS = {
-  LOGGED_USER: '/user/sessionUser/'
+  LOGGED_USER: '/user/sessionUser/',
+  USER_CRUD: '/user/users/'
```

Comment:
```suggestion
  USERS: '/user/users/'
```

Mozda i ta ruta malo deluje cudna, ocekivao bih da je samo /users/ umesto /user/users/